src/calculateRentalCost.js

```diff
@@ -4,7 +4,24 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const SHORT_TERM = 3;
+  const BIG_DISCOUNT = 50;
+  const SMALL_DISCOUNT = 20;
+  const basePrice = days * PRICE_PER_DAY;
+
+  if (days < SHORT_TERM) {
+    return basePrice;
+  }
+
+  if (days >= SHORT_TERM && days < LONG_TERM) {
+    return basePrice - SMALL_DISCOUNT;
+  }
+
+  if (days >= LONG_TERM) {
```

Comment:
Since the previous conditions return out of the function, this `if` check is not necessary. If the code execution has reached this point, it means that `days` is indeed `>= LONG_TERM`. You can remove the `if` statement and just return the calculation.